websocket/2.1/apidocs/client/constant-values.html

```diff
@@ -0,0 +1,182 @@
+<ul>
+<li><a href="#jakarta.websocket">jakarta.websocket.*</a></li>
+</ul>
+</section>
+</div>
+<div class="constantValuesContainer"><a id="jakarta.websocket">
+<!--   -->
+</a>
+<section role="region">
+<h2 title="jakarta.websocket">jakarta.websocket.*</h2>
+<ul class="blockList">
+<li class="blockList">
+<table class="constantsSummary">
+<caption><span>jakarta.websocket.<a href="jakarta/websocket/HandshakeResponse.html" title="interface in jakarta.websocket">HandshakeResponse</a></span><span class="tabEnd">&nbsp;</span></caption>
+<tr>
+<th class="colFirst" scope="col">Modifier and Type</th>
+<th class="colSecond" scope="col">Constant Field</th>
+<th class="colLast" scope="col">Value</th>
+</tr>
+<tbody>
+<tr class="altColor">
+<td class="colFirst"><a id="jakarta.websocket.HandshakeResponse.SEC_WEBSOCKET_ACCEPT">
+<!--   -->
+</a><code>public&nbsp;static&nbsp;final&nbsp;<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang" class="externalLink">String</a></code></td>
+<th class="colSecond" scope="row"><code><a href="jakarta/websocket/HandshakeResponse.html#SEC_WEBSOCKET_ACCEPT">SEC_WEBSOCKET_ACCEPT</a></code></th>
+<td class="colLast"><code>"Sec-WebSocket-Accept"</code></td>
+</tr>
+</tbody>
+</table>
+</li>
+</ul>
+</section>
+</div>
+</main>
+<footer role="contentinfo">
+<nav role="navigation">
+<!-- ======= START OF BOTTOM NAVBAR ====== -->
+<div class="bottomNav"><a id="navbar.bottom">
+<!--   -->
+</a>
+<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
+<a id="navbar.bottom.firstrow">
+<!--   -->
+</a>
+<ul class="navList" title="Navigation">
+<li><a href="jakarta/websocket/package-summary.html">Package</a></li>
+<li>Class</li>
+<li>Use</li>
+<li><a href="jakarta/websocket/package-tree.html">Tree</a></li>
+<li><a href="deprecated-list.html">Deprecated</a></li>
+<li><a href="index-all.html">Index</a></li>
+<li><a href="help-doc.html">Help</a></li>
+</ul>
+<div class="aboutLanguage">Jakarta WebSocket API v2.1.0</div>
+</div>
+<div class="subNav">
+<ul class="navList" id="allclasses_navbar_bottom">
+<li><a href="allclasses.html">All&nbsp;Classes</a></li>
+</ul>
+<div>
+<script type="text/javascript"><!--
+  allClassesLink = document.getElementById("allclasses_navbar_bottom");
+  if(window==top) {
+    allClassesLink.style.display = "block";
+  }
+  else {
+    allClassesLink.style.display = "none";
+  }
+  //-->
+</script>
+<noscript>
+<div>JavaScript is disabled on your browser.</div>
+</noscript>
+</div>
+<a id="skip.navbar.bottom">
+<!--   -->
+</a></div>
+<!-- ======== END OF BOTTOM NAVBAR ======= -->
+</nav>
+<p class="legalCopy"><small>Comments to: <a href="mailto:websocket-dev@eclipse.org">websocket-dev@eclipse.org</a>.<br> Copyright &#169; 2018, 2020 Eclipse Foundation. All rights reserved.<br> Use is subject to <a href="./doc-files/EFSL.html" target="_top">license terms</a>.</small></p>
```

Comment:
The years of copyright are "2018, 2020", but the ones of the specification documents are "2018, 2021". They are inconsistent.

Reply:
I have fixed this but there is an issue with the CI system at the moment preventing the build completing. As soon as that is fixed, I'll update API docs.